tools/debian/control

```diff
@@ -77,7 +77,6 @@ Package: cockpit-docker
 Architecture: all
 Depends: ${misc:Depends},
          docker.io (>= 1.3.0) | docker-engine (>= 1.3.0),
-         python,
```

Comment:
TBH I'd rather rely on the integration tests to spot this (and then move to Python 3) No other cockpit binary depends on python 2, and I'd rather not introduce a new Debian package that uses Python 2. Debian and Ubuntu has worked on getting rid of py 2 for many years.